config/config.ts

```diff
@@ -19,6 +19,51 @@ import {
   DISCORD_PROFILE_SERVICE_STAGING_HELP_GROUP,
   DISCORD_PROFILE_SERVICE_DEVELOPMENT_HELP_GROUP,
 } from "../src/constants/variables";
+import { config as configFromDotEnv } from "dotenv";
+
+export function loadEnv(env: env, fromWorkerEnv: boolean): env {
+  const Env: env = {
+    //if `fromWokerEnv` is true, then load from the `env` passed as argument to the function,
+    // else if `fromWokerEnv` is false, load from process.env
+    //(or set to '' if value from process.env is undefined) to avoid Error TS2322
```

Comment:
This comment is not required, 
Reason: by reading code one will understand

Reply:
ok